packages/datadog-instrumentations/src/helpers/web.js

```diff
@@ -0,0 +1,385 @@
+'use strict'
+
```

Comment:
Nothing in `datadog-instrumentations` should make reference to spans. Anything to do with spans should be done in plugins.